MinPlatformPkg/PlatformInit/Library/SecBoardInitLibNull/SecBoardInitLib.c

```diff
@@ -6,7 +6,6 @@ SPDX-License-Identifier: BSD-2-Clause-Patent
 **/
 
 #include <PiDxe.h>

```

Comment:
Given this cleans up headers, there is still a lot of clean up left to do.

1. `PiDxe.h` should not be used in a `BASE` module type / "SEC" library instance.
2. `BoardBeforeTempRamInit()` is defined in `Library/SecBoardInitLib.h`, that should be used here to confirm that function prototype matches what is defined as the public library interface (the header file)
3. It does not appear `PcdLib.h` needs to be included.
4. `BaseLib` is in `[LibraryClasses]` section of the INF but not referenced in the source files. Is it needed in the INF file?

Reply:
Sounds good. I think it'd be nice to include in these changes to keep all the cleanup together.

Reply:
Done

Reply:
edk2-platforms pr is up with the additional changes. 

https://github.com/tianocore/edk2-platforms/pull/878/files